pleskdistup/actions/extensions.py

```diff
@@ -164,3 +164,12 @@ def estimate_prepare_time(self) -> int:
 
     def estimate_revert_time(self) -> int:
         return 20
+
+
+class AssertLiteSpeedNotInstalled(action.CheckAction):
```

Comment:
This check is single-use. Propose to replace with general `AssertPleskExtensions` (like existing `AssertPleskComponents`).

Reply:
Looks like an overkill, but whatever